### PR TITLE
Add safety checks

### PR DESCRIPTION
**Pull request by RyanMacG**

### CODE REVIEW

**Comment by lukemorton** [comment]
Nice stuff man, was meaning to add this :)

Can we restructure the headings so we have a h1 at the start of the doc, then h2 and smaller across the rest? Then I think it's mergeable :)

File: team-norms/retrospectives.md [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
+ * Based on the potential causes, add another column asking for people to submit potential solutions
+ * Read these all out and discuss with the team
+ * Run the safety check again
+
+If the safety level has increased after this point, then you can run the retrospective.
 # The Prime Directive
 
 The purpose of the Prime Directive is to assure that a retrospective has the right culture to make it a positive and result oriented event. It makes a retrospective become an effective team gathering to learn and find solutions to improve the way of working.
 
+The prime directive should be read before a retrospective begins.
```
Comment (craigjbass) [inline team-norms/retrospectives.md]: :+1: :+1: :+1:

File: team-norms/retrospectives.md [hunk 2]
```diff
@@ -52,6 +63,31 @@ Teams are expected to self-organise into groups that make sense to perform retro
...
+Safety checks should be run before both retrospectives and comradrospectives, our numbering system is outlined below.
+
+5 - I’ll talk about anything
+4 - I’ll talk about almost anything; a few things might be hard
+3 - I’ll talk about some things, but others will be hard to say
+2 - I’m not going to say much, I’ll let others bring up issues
+1 - I’ll smile, claim everything is great and agree with authority figures
+0 - I'm not comfortable talking/I don't want to do this/I want to leave
+
+The facilitator should collect the team's safety numbers anonymously and work out if safety is high or low. If it is low the facilitator should hold an exercise to raise safety and work out what might be causing low safety.
```
Comment (craigjbass) [inline team-norms/retrospectives.md]: Same pen, same type of paper

Comment (craigjbass) [inline team-norms/retrospectives.md]: Or digitally / anonymously

**Comment by craigjbass** [comment]
Potentially, it might be worth summarising in bullets the accountabilities of the Retrospective Facilitator role

File: team-norms/retrospectives.md (hunk 2 above)
Comment (RyanMacG) [inline team-norms/retrospectives.md]: Yeah, I think currently it's at the discretion of the facilitator but I'll put them in as suggestions

**Pull request merged**